modules/@apostrophecms/schema/index.js

```diff
@@ -1425,8 +1425,8 @@ module.exports = {
             try {
               await convert(req, field, data, object);
             } catch (e) {
-              if (e instanceof Error) {
-                throw e;
+              if (e instanceof Error && e.message) {
```

Comment:
This isn't going to work either, but it's my fault. The error module doesn't currently set any property that doesn't exist on a generic Error, or potentially other people's third party subclasses of Error. We need to duck type this.

In the error module, please set `.aposError = true` on errors constructed by the error module. Then you can test for it here successfully. We could also include an `isOurs` method that tests it, for calling `apos.error.isOurs(error)`.

Let me know if you'd rather I did a separate PR for that first but I'm trying to speed you along.